apps/schema/static/schema/components/sensors/schema.yaml

```diff
@@ -17,6 +17,23 @@ relationships:
   allOf:
     - $ref: '#/relatedDevice'
 
+meta:
+  type: object
+  description: Meta details for this resource
+  properties:
+    dimensionality:
+      type: string
+      example: "[temperature]"
+      description: The dimensions of the sensor
```

Comment:
Can you update this description to not use the root word of what we're defining?  Me and you know what this means, but someone else reading our documentation will likely have difficulty correctly understanding what "Dimensionality" is based only in the description that it is "the dimensions of the sensor".